publisher/publisher.go

```diff
@@ -219,6 +220,7 @@ func loadConfig() config {
 	viper.BindEnv("aws_region")
 	viper.BindEnv("disable_lock")
 	viper.BindEnv("lock_retries")
+	viper.BindEnv("lock_group")
```

Comment:
I just have check the code and looks like this might be removed , and perhaps we should remove this and the if below

Reply:
yes i miss that, ok so this modification make sense